File: Backend/Core/Entities/GridConfig.cs
namespace Core.Entities
{
    /// <summary>
    /// Thực thể tương ứng với bảng pa_grid_config
    /// </summary>
    public class GridConfig
    {
        /// <summary>
        /// Khóa chính là GridKey (ví dụ: pa_salary_composition)
        /// </summary>
        public string GridKey { get; set; } = string.Empty;
        public string ConfigData { get; set; } = string.Empty;
    }
}

File: Frontend/src/models/GridConfig.js
/**
 * Model cho cấu hình Grid
 */
export default class GridConfig {
  constructor(data = {}) {
    // Khóa của Grid (ví dụ: pa_salary_composition)
    this.gridKey = data.gridKey || ''
    // Dữ liệu cấu hình (JSON string)
    this.configData = data.configData || ''
  }
}
